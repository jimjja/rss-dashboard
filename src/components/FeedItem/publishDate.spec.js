import React from 'react';
import renderer from 'react-test-renderer';
import PubDate from './publishDate';

describe('Publish date component', () => {
  it('should match snapshot', () => {
    const now = new Date();
    const props = {
      pubDate: now.toString(),
    };
    const tree = renderer.create(<PubDate {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
