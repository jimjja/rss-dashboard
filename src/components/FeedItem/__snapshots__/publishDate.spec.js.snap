// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Publish date component should match snapshot 1`] = `
<p
  style={
    Object {
      "display": "inline-block",
      "fontSize": 12,
      "margin": 0,
    }
  }
>
  2019-2-24
</p>
`;
